### PR TITLE
For IE11 (only), use core-js 3.6.5 polyfill

### PR DESCRIPTION
**Pull request by jcheng5**
Yes, the core-js version is 3.6.5, despite the files saying 3.6.4.
Source: https://unpkg.com/browse/core-js-bundle@3.6.5/

Panmirror requires a polyfill for IE11, and @jmcphers said bugs have arisen in the past due to spotty polyfill coverage for IE11.

The minified polyfill is 156KB (before gzip) so for now I'm restricting it to user agents with `/.*Trident.*/`. We may need additional, lighter polyfills for other browsers in the future--I'm not sure yet.

I injected this at the index.htm level because 1) I've forgotten everything I know about GWT, 2) it seemed advantageous to have a structural guarantee that the polyfill would be loaded before any other JS. If there is a cleaner way to do this with GWT I'm happy to revisit.

I did NOT remove any existing polyfills that this PR might have made superfluous.

### CODE REVIEW

**Comment by gtritchie** [comment]
FWIW our attempts to do fancier GWT things for loading polyfills have not worked out and we ended up retreating to sticking them into index.htm as you have done.

Example: https://github.com/rstudio/rstudio/pull/6680

File: src/cpp/core/gwt/GwtFileHandler.cpp [hunk 1]
```diff
@@ -133,11 +133,19 @@ void handleFileRequest(const FileRequestOptions& options,
...
+         vars["head_tags"] = std::string();
+      }
+
       // check for initJs
       if (!options.initJs.empty())
-         vars["head_tags"] = "<script>" + options.initJs + "</script>";
+         vars["head_tags"] = vars["head_tags"] + "<script>" + options.initJs + "</script>";
       else
-         vars["head_tags"] = std::string();
+         vars["head_tags"] = vars["head_tags"] + std::string();
```
Comment (jmcphers) [inline src/cpp/core/gwt/GwtFileHandler.cpp:148]: This looks like a no-op -- should we just omit the else?

Comment (jcheng5) [reply src/cpp/core/gwt/GwtFileHandler.cpp]: Good catch, thanks!

**Comment by jcheng5** [comment]
@jmcphers Do I just `#ifdef DEBUG` for that?

**Comment by jcheng5** [comment]
Oh or `NDEBUG`?

**Comment by jcheng5** [comment]
Actually, the minified version has sourcemaps available, so maybe it's fine for debug?

**Comment by jmcphers** [comment]
So it does! That's probably sufficient since I see it has a `//# sourceMappingURL=minified.js.map` comment.

**Comment by jcheng5** [comment]
FWIW it doesn't look to me like core-js overlaps with any of the polyfills RStudio is already using.

**Pull request merged**